CHANGELOG.md

```diff
@@ -5,7 +5,15 @@ project adheres to [Semantic Versioning](http://semver.org).
 
 ## [Unreleased (`master`)][unreleased]
 
-Nothing at the moment.
+### Changed
+
+- Changed project Sass variables to native CSS variables
+- Added variables for all font weights
+
+### Removed
+
+- Removed unused `base-z-index` variable
+- Removed unused `medium-gray` variable
```

Comment:
Agree, I'll adjust the language.

Reply:
Upon further thought, I feel like removing these w/o fanfare is fine. The named font-weights 1. provide a value to the user as the correct name and corresponding number are difficult to remember and 2. are objective truths (i.e. `bold` = `700`). These are distinct from *unused* variables that are *also* subjective (i.e. `medium-gray`). 

This might be a overly simple heuristic but I don't believe `medium-gray` provides value, whereas I think `font-weight-extra-light` does.